modules/goldbachBidAdapter.js

```diff
@@ -86,7 +86,103 @@ const NATIVE_MAPPING = {
 };
 const SOURCE = 'pbjs';
 const MAX_IMPS_PER_REQUEST = 15;
-const mappingFileUrl = 'https://acdn.adnxs-simple.com/prebid/appnexus-mapping/mappings.json';
+const TRANSLATION_MAPPING = {
+  '1': 'IAB20-3',
```

Comment:
If you extract this somewhere under `libraries` it would be re-used across the 3 adapters that need it. It'd make a difference only for those that include more than one appnexus clone in the same bundle, I don't know if that's common.

Reply:
Thanks for the suggestion, I'll look into this.